product_carousel_image_attachment/models/ir_attachment.py

```diff
@@ -2,13 +2,30 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, models
+from odoo.tools import ImageProcess
 
 IMAGE_TYPES = ["image/png", "image/jpeg", "image/bmp", "image/gif"]
 
 
 class IrAttachment(models.Model):
     _inherit = "ir.attachment"
 
+    def _resize_image(self, datas):
+        ICP = self.env["ir.config_parameter"].sudo().get_param
+        max_resolution = (
+            "1025x1025"  # Use 1025 instead of 1024 to enable the zoom feature
+        )
+        quality = int(ICP("base.image_autoresize_quality", 80))
+        img = ImageProcess(datas, verify_resolution=False)
+        w, h = img.image.size
+        nw, nh = map(int, max_resolution.split("x"))
```

Comment:
```suggestion
        max_resolution = (1025,1025)  # Use 1025 instead of 1024 to enable the zoom feature
        quality = int(ICP("base.image_autoresize_quality", 80))
        img = ImageProcess(datas, verify_resolution=False)
        w, h = img.image.size
        nw, nh = max_resolution
```
I thought splitting the text is not necessary.